Taskfile.yml

```diff
@@ -151,3 +151,40 @@ tasks:
     desc: Update local repositories to use correct hostname
     cmds:
       - go run "{{ .TASKFILE_DIR }}/common/Go/switch_repo_domains.go"
+
+  pkgconfig:
+    desc: Find which package provides a given pkgconfig. Example usage `go-task pkgconfig -- Qt5Core Qt6Core`
+    cmds:
+      - python {{ .TASKFILE_DIR }}/common/Scripts/epcsearch.py {{ .CLI_ARGS }}
```

Comment:
Note that `{{ .TASKFILE_DIR }}` is probably going to break when we go monorepo and move Taskfile.yml to the top of the monorepo. `{{ .ROOT_DIR }}/common` is probably better, though it assumes we keep all the scripts in `/common` after the monorepo is created.